packages/atkmm16.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Atkmm16 < Package
+class Atkmm16 < Meson
   description 'Atkmm is the official C++ interface for the ATK accessibility toolkit library.'
   homepage 'https://www.gtkmm.org/'
   version '2.28.3'
```

Comment:
Update to [2.36.3](https://gitlab.gnome.org/GNOME/atkmm/-/tags/2.36.3)?

Reply:
That version is in `atkmm.rb`. `atkmm16` is the version with the older API.